src/github.com/getlantern/flashlight/pac.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/getlantern/pac"
+
+	"github.com/getlantern/flashlight/proxiedsites"
+)
+
+func pacOn() {
+	if proxiedsites.PACURL != "" {
+		iconFile := ""
+		icon, err := Asset("icons/64on.ico")
+		if err != nil {
+			log.Errorf("Unable to get escalation prompt icon: %v", err)
+		} else {
+			f, err := ioutil.TempFile("", "")
```

Comment:
Fixed.  I also ended up putting it in a short path, because kAuthorizationEnvironmentIcon doesn't seem to like long paths.